### PR TITLE
Add prerequisites.yml to install steps

### PR DESCRIPTION
**Pull request by michaelgugino**
Adding reference to running one-time step of
prerequisistes.yml

### CODE REVIEW

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -2287,7 +2287,24 @@ xref:running-the-advanced-installation-individual-components[Running Individual
 The RPM-based installer uses Ansible installed via RPM packages to run playbooks
 and configuration files available on the local host. To run the installer, use
 the following command, specifying `-i` if your inventory file located somewhere
-other than *_/etc/ansible/hosts_*:
+other than *_/etc/ansible/hosts_*.
+
+
+First, running prerequisites will configure base packages and docker. It needs
```
Comment (nengard) [inline install_config/install/advanced_install.adoc]: Consider formatting to match modular templates:

Prequisites

Run `prequisites.yml` to configure the base packages and Docker.  This needs to be run only once between deploying a new cluster.

CODE HERE

Procedure

. Install OpenShift:

Comment (nengard) [reply install_config/install/advanced_install.adoc]: If reformatting is not desired, rewording is still necessary to make the first sentence clearer.

"Run `prequisites.yml` to configure the base packages and Docker.  This needs to be run only once between deploying a new cluster."

**Review by ncbaratta — CHANGES_REQUESTED** [review]

**Comment by vikram-redhat** [comment]
@adellape - PTAL for BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1525357#c12

**Comment by adellape** [comment]
Taking care of this via https://github.com/openshift/openshift-docs/pull/8205.

**Pull request closed without merging**